### DISCUSSION TITLE
Cant get curio items to work in my mod

### DISCUSSION THREAD

**Discussion by SucksAtstuff** [TheIllusiveC4/Curios#432] [Q&A]
I am trying to create a curio item. I can make the slots appear, but I can't get the item to be equippable in the game. Any help would be appreciated.

All my code is here: https://github.com/SucksAtstuff/Succs-Mod

**Comment by TheIllusiveC4** [comment]
It looks like you're missing the curio tags on your items. Check out this page for more info:
https://docs.illusivesoulworks.com/curios/items/assign-slots

**Reply by SucksAtstuff** [reply]
im looking at it but cant figure out how to add the tags

**Reply by TheIllusiveC4** [reply]
They go into your mod's datapack, like other tags. Specifically, it would be in `data/curios/tags/items/<slot>.json` with `<slot>` being replaced by the identifier for your slot.

**Reply by SucksAtstuff** [reply]
ah thank you it works now :D

**Reply by SucksAtstuff** [reply]
actually now that i already got you here ive figured out how to add extra slots to the player but how would i add two of the same slot type
